ansible/openstack-ratings.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Generate OpenStack software ratings
+  hosts: localhost
+  tags:
+    - ratings
+  roles:
+    - role: stackhpc.openstack.os_ratings
+      os_ratings_hashmap_field_mappings: "{{ ratings_hashmap_field_mappings }}"
+      os_ratings_hashmap_service_mappings: "{{ ratings_hashmap_service_mappings }}"
+      os_ratings_venv: "{{ openstack_venv }}"
+      os_ratings_upper_constraints_file: "{{ ratings_upper_constraints_file }}"
```

Comment:
Can drop overriding ``os_ratings_upper_constraints_file``, the default is sufficient.

Reply:
I still think providing the option is better than locking it. Especially as this is going to be where CloudKitty is almost solely going to be configured.

Reply:
I was thinking along the lines of matching the conventions the other roles use. I can't immediately see why we'd want to override these values often, but I guess that it doesn't hurt to expose them.